frontend/src/components/Database/DatabaseChangeHistoryPanel.vue

```diff
@@ -38,6 +38,21 @@
             </div>
           </template>
         </TooltipButton>
+        <TooltipButton
+          v-if="allowAlterSchema"
+          tooltip-mode="DISABLED-ONLY"
+          :disabled="state.selectedChangeHistoryNameList.length !== 1"
```

Comment:
Only DDL and baseline changehistory can be rolled back to.

DML is not.

Reply:
Updated